crates/swc_ecma_transforms_typescript/src/strip.rs

```diff
@@ -594,12 +594,6 @@ where
                         if let Some(Some(v)) = values.get(&id.sym) {
                             return Ok(v.clone());
                         }
-                        //
```

Comment:
cc @magic-akari Do you think this is fine?

Reply:
All test passed. It's OK in my opinion.